Main.cpp

```diff
@@ -252,6 +252,34 @@ void _LoadingSplashProgressReport(std::string reportString, bool newItem = false
 
 void LoadingSplashProgressReport(std::string reportString, bool newItem = false)
 {
+	if(g_LogToCli)
+	{
+		if (newItem)
+		{
+			std::cout << std::endl;
+		}
+		// overwrite current line
+		std::cout << "\r";
+		size_t startPos = 0;
+		// convert characters to unicode
+		// just make sure to really overwrite all old output
+		std::string unicoded = reportString + "          ";
```

Comment:
I think the comment explaining what this does is good enough, but I'm personally curious why you have to add a bunch of spaces at the end, would you mind explaining it to me in discord? Thanks.